templates/cli/lib/config.js.twig

```diff
@@ -8,7 +8,8 @@ const KeysFunction = new Set(["path", "$id", "execute", "name", "enabled", "logg
 const KeysDatabase = new Set(["$id", "name", "enabled"]);
 const KeysCollection = new Set(["$id", "$permissions", "databaseId", "name", "enabled", "documentSecurity", "attributes", "indexes"]);
 const KeysStorage = new Set(["$id", "$permissions", "fileSecurity", "name", "enabled", "maximumFileSize", "allowedFileExtensions", "compression", "encryption", "antivirus"]);
-const KeyTopics = new Set(["$id", "name", "subscribe"]);
+const KeysTopics = new Set(["$id", "name", "subscribe"]);
+const KeysTeams = new Set(["$id", "name"]);
 const KeyAttributes = new Set([
```

Comment:
Can we also rename KeyAttributes to KeysAttributes? Seems only one without plural